lib/fizzy/capi.cpp

```diff
@@ -9,6 +9,20 @@
 #include <fizzy/fizzy.h>
 #include <memory>
 
+namespace
+{
+fizzy_instance* wrap(std::unique_ptr<fizzy::Instance> x)
```

Comment:
I think it is better not to hake `unique_ptr` because the `unwrap` cannot return one.